### PR TITLE
Hotfix/peru 124 scatterplot range

### PR DESCRIPTION
**Pull request by romsson**
screen shot 2016-04-15 at 3 09 33 pm
screen shot 2016-04-15 at 3 10 02 pm

### CODE REVIEW

File: app/templates/visualization.hbs [hunk 1]
```diff
@@ -29,6 +29,7 @@
       {{component visualizationComponent
         classNames='buildermod__viz'
         immutableData=immutableData
+        modelData=modelData
```
Comment (makmanalp) [inline app/templates/visualization.hbs]: So this uses both modelData and immutableData?

Comment (makmanalp) [inline app/templates/visualization.hbs]: So this uses both modelData and immutableData?

Comment (romsson) [inline app/templates/visualization.hbs]: `immutableData` are filtered data and is the original data being used. I've added `modelData` to figure out some global dataset attributes properties (like axis domains) on the fly by the viz component. Those properties could have been found by the controller but would have been very attribute specific (e.g. min/max for distance, complexity, ..).

**Comment by makmanalp** [comment]
👍  looks great!

**Pull request merged**